src/main/java/com/ldtteam/blockout/Pane.java

```diff
@@ -344,12 +352,33 @@ public final void drawLast(final int mx, final int my)
         if (visible)
         {
             drawSelfLast(mx, my);
+
+            if (isHovered && toolTipLines != null && !toolTipLines.isEmpty())
```

Comment:
just initialize toolTipLines with an empty list so we don't have to do both checks here